integration_test/scripts/build-adapters.sh

```diff
@@ -1,27 +1,17 @@
 #!/bin/bash
-# imageProject commitShortSha
 
-base_name="gcr.io/${1}/healthcare-api-dicom-dicomweb-adapter-"
+export PATH=$PATH:/opt/gradle/gradle-6.7/bin
 
-build_adapter () {
-  gradle dockerBuildImage
-  adapter_name=${base_name}${1}
-  version=$(gradle printVersion  | grep -Eo "[0-9]{1,3}\.[0-9]{1,3}" | head -1)
-  postfix=$(gcloud container images list-tags "${adapter_name}" --sort-by=~timestamp | grep " $version" |  grep -Eo "([0-9]{1,3}\.){2}[0-9]{1,3}" | head -1 | grep -Eo "[0-9]{1,3}" | tail -1)
-  postfix=$(($postfix+1))
-  gradle dockerBuildImage -Pdocker_tag=${adapter_name}:$version.$postfix
-  gradle dockerBuildImage -Pdocker_tag=${adapter_name}:$2
-}
-apt-get -qq update
+apt-get -qq update &&
 apt-get install -y gnupg2
 # we need gcloud and 5.6-jdk11 in same image next 4 lines adds google's package storage and installs gcloud by this manual https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu
 echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
 curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
 apt-get -qq update
-apt-get install google-cloud-sdk -y
+apt-get install google-cloud-sdk -y &&
 
+gradle build &&
+cd ./import &&
+gradle build &&
+cd ../export &&
```

Comment:
The build and launch of the adapter are separate, since we launch the adapter in the first step basic-test, and build the adapter in the last step publish. Duplicated part of google-cloud-sdk installation as it doesn't persist between steps